docs/contributing/coding-style-guide.md

```diff
@@ -579,7 +579,7 @@ implicit and thus more prone to surprises.
 
 #### Clean Defer Only if Function Fails
 
-There is a way to sacrifice defer in order to properly close all on each error. Repetitions makes it easier to make error
+There is a way to sacrifice defer in order to properly close all on each error. Repetitions make it easier to make an error
```

Comment:
hmm actually I think this reads more natural/native if we say `Repetition makes it easier...` i.e. in the singular

Reply:
That sounds right 👍